BuildScript/BuildTasks/MetadataValueDictionary.cs

```diff
@@ -52,9 +46,9 @@ public IEnumerable<string> GetKeys (string value)
           .Select (kvp => kvp.Key);
     }
 
-    public bool ContainsValue (string value)
+    public bool ContainsKey (string key)
     {
-      return GetKeys (value).Count() != 0;
+      return _rawDictionary.Keys.Contains (key, StringComparer.OrdinalIgnoreCase);
```

Comment:
the dictionary already uses ordinalignorecase. should be able to skip that. also, do we have tests that show that this is case-insensitive?

Reply:
Yes, you're completely right. And yes, the test is `MetadataValueDictionaryTest.ContainsKey_IgnoresCase`